### ISSUE TITLE
[BUG] Map pins not shared amongst players on the server

### ISSUE THREAD

**Issue by lennyxc** [valheimPlus/ValheimPlus#732]
- **OS:** Windows Client connecting to Linux Server
- **Valheim Game Version:** 0.212.7
- **V+ Mod Version:** 0.9.9.11

**Describe the bug**

Map pins do not seem to be shared amongst the players on the server

**To Reproduce**

1. Join dedicated server
2. Player A sets some pins on map
3. Player B sets some pins on map
4. Player A cannot see B's pins, and vice versa.

**Expected behavior**

Shared map pins visible on the map for all players.

**Troubleshooting**

What steps have you taken to try to fix?

N/A

**Screenshots**

N/A

**Config**

; Change false to true to enable this section. https://valheim.plus/documentation/list#Map
enabled = true

; With this enabled you will receive the same exploration progression as other players on the server.
; This will also enable the option for the server to sync everyones exploration progression on connecting to the server.
shareMapProgression = true

; The radius of the map that you explore when moving.
exploreRadius = 100

; Prevents you and other people on the server to turn off their map sharing option.
preventPlayerFromTurningOffPublicPosition = true

; This option automatically shares created pins with everyone playing on the server.
shareAllPins = true

; Display carts and boats on the map
displayCartsAndBoats = true

**Additional context**

*Please list any other mods besides V+ that you have installed* here. Add any other context about the problem here.

No other mods installed

**Comment by andres-valdes** [comment]
Yep. There also used to be a UI that popped up for making pins, doesn't show up either.

**Comment by snips86x** [comment]
Map pins were retired use the cartography table.

**Comment by andres-valdes** [comment]
Were they actually? The mod's implementation is infinitely better than the cartography table.

**Comment by snips86x** [comment]
Yeah, it's been the case since 9.9.7 or 9.9.8.

**Comment by SerpentDrago** [comment]
from my understanding the Pop up ui for making pins was retired , not the sharing of the pins / syncing them between players . 

other wise why would it be in the config file as a option ?

**Comment by SSimp258** [comment]
> from my understanding the Pop up ui for making pins was retired , not the sharing of the pins / syncing them between players .
> 
> other wise why would it be in the config file as a option ?

Unfortunately, when you take away the GUI to add a pin, you also take away the part that shares the pin. That whole section of code was retired. From what I took away was NX was gonna come back to it, but the cartography table does almost the exact same thing.

It's been in the CFG because it's just never been removed from that section just in case NX does come back to it. The coding in V+ also hasn't been removed, it's just disabled because the GUI needed a rework to make it simple.

Right now, the "workaround," as some have called it, is to use the cartography table until when/if this feature makes a comeback.

**Comment by SerpentDrago** [comment]
yeh problem with the table is its not instant . we like using markers while out and about . to make easy markers for reference .

**Comment by andres-valdes** [comment]
> > from my understanding the Pop up ui for making pins was retired , not the sharing of the pins / syncing them between players .
> > other wise why would it be in the config file as a option ?
> 
> Unfortunately, when you take away the GUI to add a pin, you also take away the part that shares the pin. That whole section of code was retired. From what I took away was NX was gonna come back to it, but the cartography table does almost the exact same thing.
> 
> It's been in the CFG because it's just never been removed from that section just in case NX does come back to it. The coding in V+ also hasn't been removed, it's just disabled because the GUI needed a rework to make it simple.
> 
> Right now, the "workaround," as some have called it, is to use the cartography table until when/if this feature makes a comeback.

Do you have the issue/PR where this was discussed? I'd love to see the discussion and possibly contribute a fix.

**Comment by pdickey** [comment]
#672 is probably the best place for it before **SSimp258** derailed the thread by repeatedly saying that the feature was not coming back and to just use the cartography table instead.

I submitted the issue pretty quickly after the change went live, so you can use the date and version info to start digging in.

**Comment by LetoDK** [comment]
Yup, SSimp258 has a weird "chip on his shoulder" about this topic, which makes him spread the misinformation that the feature "has been retired", when the developers have explicitly stated on multiple occasions that the feature is not retired, but simply currently broken.

@pdickey I would like to kindly remind you that it's probably a good idea to reopen [your issue report](https://github.com/valheimPlus/ValheimPlus/issues/672)  at some point, as it's currently set to "closed".

**Comment by SSimp258** [comment]
> Yup, SSimp258 has a weird "chip on his shoulder" about this topic, which makes him spread the misinformation that the feature "has been retired", when the developers have explicitly stated on multiple occasions that the feature is not retired, but simply currently broken.

Not a chip if this is the information I've been told by Devs and by other Chat moderators on the discord. You guys have had another chat mod come in here and confirm it so i'm asking you guys to cease and desist the incorrect messages you are posting. If you don't, I will ask NX to block you from this GIT.

If you decide to reopen the issue, that's fine but it's not high on the fix list unless someone else wants to try to fix it. That's the other thing NX has said and that's why it was retired. See my other messages above if you guys want to fix it yourself. I gave you a roadmap and a starting point in a previous message.

**Comment by pdickey** [comment]
> @pdickey I would like to kindly remind you that it's probably a good idea to reopen [your issue report](https://github.com/valheimPlus/ValheimPlus/issues/672) at some point, as it's currently set to "closed".

Sure. #672 has more history so I'll reopen it, but that means that this issue is a duplicate.

**Comment by LetoDK** [comment]
@pdickey Great stuff👍 I believe the devs can merge the two issue reports if they need to.
